### PR TITLE
replaced incorrect info

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6892

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/client-management/mdm/policy-csp-restrictedgroups.md [hunk 1]
```diff
@@ -137,17 +137,17 @@ Here's an example:
...
     </accessgroup>
 </groupmembership>
 ```
 where:
 - `<accessgroup desc>` contains the local group SID or group name to configure. If an SID is specified here, the policy uses the [LookupAccountName](https://docs.microsoft.com/windows/win32/api/winbase/nf-winbase-lookupaccountnamea) API to get the local group name. For best results, use names for `<accessgroup desc>`.
 - `<member name>` contains the members to add to the group in `<accessgroup desc>`. If a name is specified here, the policy will try to get the corresponding SID using the [LookupAccountSID](https://docs.microsoft.com/windows/win32/api/winbase/nf-winbase-lookupaccountsida) API. For best results, use SID for `<member name>`. The member SID can be a user account or a group in AD, Azure AD, or on the local machine. Membership is configured using the [NetLocalGroupSetMembers](https://docs.microsoft.com/windows/win32/api/lmaccess/nf-lmaccess-netlocalgroupsetmembers) API.
-- In this example, `Group1` and `Group2` are local groups on the device being configured.
+- In this example, `Group1` and `Group2` are local groups on the device being configured, and `Group3` - is a domain group.
 
 > [!Note]
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-restrictedgroups.md:149]:
```suggestion
> [!NOTE]
```

File: windows/client-management/mdm/policy-csp-restrictedgroups.md [hunk 2]
```diff
@@ -137,17 +137,17 @@ Here's an example:
...
-> Currently, the RestrictedGroups/ConfigureGroupMembership policy does not have a MemberOf functionality. However, you can add a local group as a member to another local group by using the member portion, as shown in the above example.
+> Currently, the RestrictedGroups/ConfigureGroupMembership policy does not have a MemberOf functionality. However, you can add a domain group as a member to a local group by using the member portion, as shown in the above example.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-restrictedgroups.md:150]:
```suggestion
> Currently, the RestrictedGroups/ConfigureGroupMembership policy does not have a MemberOf functionality. However, you can add a domain group as a member to a local group by using the member portion, as shown in the previous example.
```

File: windows/client-management/mdm/policy-csp-restrictedgroups.md [hunk 3]
```diff
@@ -137,17 +137,17 @@ Here's an example:
...
     </accessgroup>
 </groupmembership>
 ```
 where:
 - `<accessgroup desc>` contains the local group SID or group name to configure. If an SID is specified here, the policy uses the [LookupAccountName](https://docs.microsoft.com/windows/win32/api/winbase/nf-winbase-lookupaccountnamea) API to get the local group name. For best results, use names for `<accessgroup desc>`.
 - `<member name>` contains the members to add to the group in `<accessgroup desc>`. If a name is specified here, the policy will try to get the corresponding SID using the [LookupAccountSID](https://docs.microsoft.com/windows/win32/api/winbase/nf-winbase-lookupaccountsida) API. For best results, use SID for `<member name>`. The member SID can be a user account or a group in AD, Azure AD, or on the local machine. Membership is configured using the [NetLocalGroupSetMembers](https://docs.microsoft.com/windows/win32/api/lmaccess/nf-lmaccess-netlocalgroupsetmembers) API.
-- In this example, `Group1` and `Group2` are local groups on the device being configured.
+- In this example, `Group1` and `Group2` are local groups on the device being configured, and `Group3` - is a domain group.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-restrictedgroups.md:147]:
```suggestion
- In this example, `Group1` and `Group2` are local groups on the device being configured, and `Group3` is a domain group.
```

**Comment by ManikaDhiman** [comment]
Changes reviewed and approved by the policy owner; hence, merging this PR. Thanks!

**Pull request merged**